config/slim-lint.yml

```diff
@@ -10,23 +10,30 @@ linters:
   RuboCop:
     # TODO: Remove (and let defaults apply) once https://github.com/sds/slim-lint/issues/30 is fixed
     ignored_cops:
-      - Style/IndentHash
+      - Layout/IndentHash
       # defaults from slim-lint:
+      - Layout/AlignHash
+      - Layout/AlignParameters
+      - Layout/FirstParameterIndentation
+      - Layout/IndentationConsistency
+      - Layout/IndentationWidth
+      - Layout/MultilineArrayBraceLayout
+      - Layout/MultilineAssignmentLayout
+      - Layout/MultilineHashBraceLayout
+      - Layout/MultilineMethodCallBraceLayout
+      - Layout/MultilineMethodCallIndentation
+      - Layout/MultilineMethodDefinitionBraceLayout
+      - Layout/MultilineOperationIndentation
+      - Layout/TrailingBlankLines
+      - Layout/TrailingWhitespace
       - Lint/BlockAlignment
       - Lint/EndAlignment
       - Lint/Void
-      - Metrics/LineLength
       - Metrics/BlockLength
-      - Style/AlignHash
-      - Style/AlignParameters
+      - Metrics/LineLength
       - Style/BlockNesting
       - Style/FileName
-      - Style/FirstParameterIndentation
       - Style/FrozenStringLiteralComment
       - Style/IfUnlessModifier
-      - Style/IndentationConsistency
-      - Style/IndentationWidth
       - Style/Next
-      - Style/TrailingBlankLines
-      - Style/TrailingWhitespace
```

Comment:
These are the new defaults copied from `slim-lint` plus `Layout/IndentHash` which we already had.